INSTALL.md

```diff
@@ -0,0 +1,67 @@
+# How to install MacGitver
+
+In present state, you need to compile MacGitver from source. We provide a compilable version from our default branch `development`. If you like to help with creating binary packages, please create a [pull request](https://github.com/macgitver/MacGitver/pulls) or contact me via [mail](mailto:nils@macgitver.org).
+
+## Prerequisites
```

Comment:
@scunz FYI: The Qt installer greatly changed (for the better really), so Qt and Qt-Creator are both not installed by apt-get, but via the Qt-Maintenance tool (which is installed by online-installer). The official Linux binaries are compiled with GCC, but I never had any problems with mixing GCC and Clang in the past 3 years of MGV development :). For a "simple user" I always advice to actually use the official installers provided by the Qt-Company. By default, this installs Qt and Qt-Creator to `$HOME/Qt` ... and there you'll find all the "-dev" libraries :).